eclipse-collections-api/src/main/java/org/eclipse/collections/api/bag/Bag.java

```diff
@@ -116,6 +116,26 @@
      */
     Bag<T> selectByOccurrences(IntPredicate predicate);
 
+    /**
+     * Returns all elements of the bag that have exactly one occurrence.
+     *
+     * @since 9.2.0
+     */
+    default Bag<T> selectByUniques()
+    {
+        return this.selectByOccurrences(occurrences -> occurrences == 1);
+    }
+
+    /**
+     * Returns all elements of the bag that have more than one occurrence.
+     *
+     * @since 9.2.0
+     */
+    default Bag<T> selectByDuplicates()
+    {
+        return this.selectByOccurrences(occurrences -> occurrences > 1);
+    }
+
```

Comment:
Let's talk about naming.

* In the commit messages and PR, selectUniques and selectDuplicates. SelectDuplicates sounds ok, but selectUniques sounds a little awkward. Maybe selectUnique?
* In the code, selectByUniques and selectByDuplicates. The plural is definitely awkward. Maybe selectByUnique and selectByDuplicate?
* Some other similar choices, selectUnique/selectDuplicate? selectDuplicates/rejectDuplicates?

Reply:
I'll make the commit message and method names consistent.
Having plural is awkward I agree, but saying selectByUnique might allude to someone that they are getting only 1 element back. That said I don't really have strong feelings about any of the names.

Reply:
I think selectByUnique and selectByDuplicate... it's a shorthand for selectByUniqueOccurrences and selectByDuplicateOccurrences.

Reply:
I think "By" is a stretch. I think conversationally, it would be more natural to drop "By". Also, our APIs are pretty consistent today in that By means the method takes a function, which normally returns an attribute or some calculated value to go by.

My ranked preferences:
* selectDuplicates/rejectDuplicates
* selectUnique/selectDuplicate
* selectUniqueOccurrences/selectDuplicateOccurrences

Reply:
How about selectDuplicates (plural), selectUnique  (singular)
In the Javadoc we can clarify to say duplicates returns a bag of all repeating elements and unique returns a bag of all unique elements